test/integration/node-specific/resource_clean_up.test.ts

```diff
@@ -0,0 +1,108 @@
+import { fork } from 'node:child_process';
+import { on, once } from 'node:events';
+import { readFile, unlink, writeFile } from 'node:fs/promises';
+import * as path from 'node:path';
+
+import { expect } from 'chai';
+import { parseSnapshot } from 'v8-heapsnapshot';
+
+const HEAPSNAPSHOT_BEFORE = './before.heapsnapshot.json';
+const HEAPSNAPSHOT_AFTER = './after.heapsnapshot.json';
+
+const REPRO_SCRIPT = uri => `
+const { MongoClient } = require('${path.resolve(__dirname, '../../../lib')}');
```

Comment:
```suggestion
const { MongoClient } = require(${JSON.stringify(path.resolve(__dirname, '../../../lib'))});
```

(would always do this when building template strings containing JS codes, handles all the edge cases like "what if `__dirname` includes a forward slash" which is extremely unlikely here, but, you know 🙂 )

Reply:
Good call, I was thinking this or inspect, but that would just be for getting single quotes